ecl/eclagent/eclagent.ipp

```diff
@@ -512,6 +513,20 @@ public:
     {
         return queryNullSectionTimer();
     }
+// IEngineContext
+    virtual DALI_UID getGlobalUniqueIds(unsigned num, SocketEndpoint *_foreignNode) { throwUnexpected(); }
```

Comment:
@richardkchapman unfortunately this may also now need to be implemented - see fsGetUniqueInteger in fileservices.esp, which would have previously worked for hthor (and sometimes in roxie).